Support/Multitenancy/Scalability-with-Fake-Nodes/bmo-config/kustomization.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: baremetal-operator-system
+resources:
+- https://github.com/metal3-io/baremetal-operator/config/overlays/basic-auth_tls?ref=release-0.6
+
+configMapGenerator:
+- name: ironic
+  behavior: create
+  envs:
+  - ironic.env
+
+patches:
+- patch: |
+    # Don't try to pull again the pre-loaded image
+    - op: replace
+      path: /spec/template/spec/containers/0/imagePullPolicy
+      value: IfNotPresent
+  target:
+    kind: Deployment
+    name: controller-manager
+
+images:
+- name: quay.io/metal3-io/baremetal-operator
+  newTag: release-0.5
```

Comment:
Why release-0.5 here but release-0.6 above?

Reply:
I guess I changed it at some point but forgot to update? I'll fix that, thanks :D